local.c

```diff
@@ -1183,7 +1183,7 @@ static int read_device_name(struct iio_device *dev)
 
```

Comment:
nitpick: comment title is a bit long; it could be trimmed to `local: Sort devices, channels and attributes when adding them`

git-style comments prefer an empty between title & description

Reply:
ah; and the description shouldn't be indented ;